src/runtime_src/core/common/drv/include/kds_core.h

```diff
@@ -58,11 +58,13 @@ struct kds_ctx_info {
  * Let's see if we can unify them later.
  */
 struct kds_scu_mgmt {
+	struct xrt_cu		 *xcus[MAX_CUS];
```

Comment:
Maybe we need another struct for PS kernel. But right now, subdev/scu.c is not really talking to PS, so this is okay to me.